tony-examples/tony-in-gcp/README.md

```diff
@@ -0,0 +1,524 @@
+```
+
+#### Connect via SSH
+
+Go to  Cloud Console -> DataProc -> Big Data -> Clusters 
+
+![img](https://i.imgur.com/7NdNYja.png)
+
+**Connect SSH to DataProc master server:**
+
+Click on SSH and connect remotely to Master server.
+
+**Verify Operating System version:**
+
+Google Cloud Dataproc installs a Debian version 9. This can be modified via `image-version` parameter.
+
+```
+uname -a
+Linux tony-staging-m 4.9.0-8-amd64 #1 SMP Debian 4.9.110-3+deb9u6 (2018-10-08) x86_64 GNU/Linux
+```
+```
+cat /etc/debian_version
+9.5
+```
+```
+cat /etc/issue
+Debian GNU/Linux 9 \n \l
+```
+```
+cat /etc/os-release
+PRETTY_NAME="Debian GNU/Linux 9 (stretch)"
+NAME="Debian GNU/Linux"
+VERSION_ID="9"
+VERSION="9 (stretch)"
+ID=debian
+HOME_URL="https://www.debian.org/"SUPPORT_URL="https://www.debian.org/support"
+BUG_REPORT_URL="https://bugs.debian.org/"
+```
+
+**Verify Hadoop, HDFS and YARN versions**
+
+```
+hadoop version
+```
+
+Example output
+
+```
+Hadoop 2.9.0 Subversion 
+https://bigdataoss-internal.googlesource.com/third_party/apache/hadoop -r e8ce80c37eebb173fc688e7f5686d7df74d182aa
+Compiled by bigtop on 2018-10-25T12:56Z
+Compiled with protoc 2.5.0From source with checksum 1eb388d554db8e1cadcab4c1326ee72
+This command was run using /usr/lib/hadoop/hadoop-common-2.9.0.jar
+```
+
+**Verify Java installation**
+
+Java version 1.8, is installed by default. In order to verify you have Java version > 1.8 run:
+
+```
+java -version
+```
+
+Example:
+
+```
+openjdk version "1.8.0_171"OpenJDK Runtime Environment (build 1.8.0_171-8u171-b11-1~bpo8+1-b11)
+OpenJDK 64-Bit Server VM (build 25.171-b11, mixed mode))
+```
+
+**Verify JAVA JDK**
+
+```
+echo $JAVA_HOME
+```
+
+Verify both are matching JAVA 1.8, by default $JAVA_HOME is set to version 1.8, if not, change it to Java version 1.8 using:
+
+```
+sudo update-alternatives --config java
+```
+
+**Verify YARN nodes are active**
+
+```
+yarn node -list
+```
+
+Example
+
+```
+yarn node -list
+
+18/11/19 00:48:23 INFO client.RMProxy: Connecting to ResourceManager at tony-staging-m/10.138.0.2:8032
+18/11/19 00:48:24 INFO client.AHSProxy: Connecting to Application History server at tony-staging-m/10.138.0.2:10200
+Total Nodes:2         Node-Id             Node-State Node-Http-Address       Number-of-Running-Containers
+tony-staging-w-0.c.dpe-cloud-mle.internal:39349         RUNNING tony-staging-w-0.c.dpe-cloud-mle.internal:8042                             0
+tony-staging-w-1.c.dpe-cloud-mle.internal:44617         RUNNING tony-staging-w-1.c.dpe-cloud-mle.internal:8042                             0
+```
+
+**Connect to Cloud Shell**
+
+Open cloud shell via Pantheon UI
+
+![img](https://i.imgur.com/3FIIk5Y.png)
+
+**Verify PySpark**
+
+Submit a PySpark to verify installation.
+
+```
+gcloud dataproc jobs submit pyspark --cluster tony-staging --region global \
+gs://dataproc-examples-2f10d78d114f6aaec76462e3c310f31f/src/pyspark/hello-world/hello-world.py
+```
+
+**Verify MapReduce job (Optional)**

```

Comment:
What's the motivation for doing this verification?

Reply:
Just to confim if Dataproc cluster deployment was successful